proposals/lip-0045.md

```diff
@@ -391,7 +391,7 @@ The channel data substore holds information about the inbox and outbox with othe
 ```java
 channelDataSchema = {
     "type": "object",
-    "required": ["inbox", "outbox", "partnerChainOutboxRoot", "messageFeeTokenID"],
+    "required": ["inbox", "outbox", "partnerChainOutboxRoot", "messageFeeTokenID", "minReturnFeePerByte"],
```

Comment:
unfortunate consequence of adding an extra property: Figure 2 should also be updated..

Reply:
updated